server/routes/courtRegister/courtMapper.ts

```diff
@@ -58,39 +58,39 @@ export function courtBuildingMapper(building: CourtBuilding): CourtBuildingDetai
   return {
     id: building.id,
     code: building.subCode,
-    name: building.buildingName,
+    name: building.buildingName as string,
     addressline1: building.street,
     addressline2: building.locality,
     addresstown: building.town,
     addresscounty: building.county,
     addresspostcode: building.postcode,
     addresscountry: building.country,
-    contacts: building.contacts.map(courtBuildingContactMapper),
```

Comment:
contacts can not be - but swagger thinks it can - might need to change court-register